doc/services.rst

```diff
@@ -29,6 +31,21 @@ Amazon DynamoDB
 
 .. autoclass:: stepfunctions.steps.service.DynamoDBUpdateItemStep
 
+
+Amazon EKS
+----------
+.. autoclass:: stepfunctions.steps.service.EksCreateClusterStep
+
+.. autoclass:: stepfunctions.steps.service.EksCreateFargateProfileStep
+
+.. autoclass:: stepfunctions.steps.service.EksCreateNodeGroupStep
```

Comment:
`Nodegroup` in EKS speak (not camel cased) - https://docs.aws.amazon.com/eks/latest/APIReference/API_CreateNodegroup.html
```suggestion
.. autoclass:: stepfunctions.steps.service.EksCreateNodegroupStep
```

Reply:
Good catch! will make all the name changes in the same commit!